File: AtividadeHerancaJs/Professor.js
class Professor extends Pessoa{
    constructor(nome, idade, disciplina){
        super(nome, idade);
        this.disciplina=disciplina;
    }

    ensinar(){
        return`
        Nome:${this.nome}
        Idade:${this.idade}
        Disciplina:${this.disciplina}
        `;
    }
}

const pessoa2 = new Pessoa(
    "Tiago",
    32
)

const professor1 = new Professor(
    "Tiago",
    32,
    "Matemática"
)

File: AtividadeHerancaJava/AtividadeHerancaJava/Professor.java
public class Professor extends Pessoa{
    private String Disciplina;
    private String Ensinar;

    public Professor(String Nome, int Idade, String Disciplina, String Ensinar){
        super(Nome, Idade);
        this.Disciplina = Disciplina;
        this.Ensinar = Ensinar;
    }
    @Override
    public void estudar(){
        System.out.println("====ALUNO====");
        System.out.println("Nome:" + Nome);
        System.out.println("Idade:" + Idade);
        System.out.println("Disciplina:" + Disciplina);
        System.out.println("Ensinar:" + Ensinar);
    }
}
